bus-mapping/src/evm/opcodes/callop.rs

```diff
@@ -51,6 +57,11 @@ impl<const N_ARGS: usize> Opcode for CallOpcode<N_ARGS> {
                 CallContextField::CalleeAddress,
                 current_call.address.to_word(),
             ),
+            (
```

Comment:
only when current call is delegate call need this , suggest to add if condition statement .

Reply:
Fixed to lookup parent caller address and parent value only for opcode `DELEGATECALL`.